core/domain/stats_services.py

```diff
@@ -77,7 +91,12 @@ def get_versions_for_exploration_stats(exploration_id):
 def get_exploration_stats(exploration_id, exploration_version):
     """Returns a dict with state statistics for the given exploration id.
 
-    Note that exploration_version should be a string.
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: str. The version of the exploration.
```

Comment:
Hi @manoj-makkuboy -- I'm not sure I understand, sorry. Can you explain what sequence of steps led you to get "all" as the version? Similarly, can you explain what sequence of steps led you to get an int?